### PR TITLE
Bug/save cart

### PR DESCRIPTION
**Pull request by Kratika0907**
Resolved conflicts
Added feature for save later and delete items
CSS changes for cart page

### CODE REVIEW

File: redux/action.js [hunk 1]
```diff
@@ -34,11 +37,41 @@ export const addShopListItem = (item) => {
...
+
+// export const moveToCart = (item) => {
+//   return {
+//     type : MOVE_TO_CART,
+//     payload: {
+//       item
+//     }
+//   }
+// }
+
```
Comment (whyDontI) [inline redux/action.js:68]: Remove these comments if not needed

**Review by whyDontI — CHANGES_REQUESTED** [review]

File: redux/action.js (hunk 1 above)
Comment (Kratika0907) [reply redux/action.js:68]: They are needed for other feature that I have not pushed

**Review by whyDontI — APPROVED** [review]
LGTM 👍🏽

File: components/save-later/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+import colors from '../../color/color.json';
+const SaveLater = (props) => {
+  const { details: product } = props;
+  return (
+    <div className="savelateritem-container">
+      <img
+        className="savelater-image"
+        src={path.join('./assets', product.image)}
```
Comment (ankushdharkar) [inline components/save-later/index.js:9]: @whyDontI Why are we not using Next.js way?

Comment (ankushdharkar) [reply components/save-later/index.js:9]: cc @Kratika0907

**Pull request merged**